src/Hng.Web/Controllers/ProfileController.cs

```diff
@@ -48,10 +47,9 @@ public async Task<IActionResult> UpdateProfile([FromBody] UpdateProfileDto profi
         [HttpPut("{email}/picture")]
         [ProducesResponseType(typeof(UpdateProfilePictureResponseDto), StatusCodes.Status200OK)]
         [ProducesResponseType(typeof(UpdateProfilePictureResponseDto), StatusCodes.Status400BadRequest)]
-        public async Task<IActionResult> UpdateProfilePicture([FromForm] UpdateProfilePictureDto profileDto, string email)
+        public async Task<IActionResult> UpdateProfilePicture([FromForm] UpdateProfilePicture profileDto, string email)
         {
```

Comment:
You should be using a Dto here not the entity. Did you change the class name?

Reply:
That is a DTo, just didn't append dto to it

Reply:
> That is a DTo, just didn't append dto to it

Please append it, for clarity purposes.

Reply:
ok